State.x

```diff
@@ -25,6 +26,27 @@ static NSString *stateFile = ROOT_PATH_NS(@"/var/mobile/.beepserv_state");
 	return [NSString stringWithFormat:@"<BPState code:%@ secret:%@ connected:%d error:%@", self.code, self.secret, self.connected, self.error];
 }
 
+- (void)broadcast {
+	NSMutableDictionary *state = @{
```

Comment:
Since this struct is serialized to a dictionary multiple times, could we extract that into a method so we can reuse it in both the 'broadcast' and 'writeToDisk' methods?

Reply:
Yes, I've created a new `serializeToDictionary ` method and moved it in there.